doc/docs/master/how-tos/export-data-out-pachyderm/mount-repo-to-local-computer.md

```diff
@@ -0,0 +1,187 @@
+# Mount a Repo to a Local Computer
+
+Sometimes you might need to access files directly in
+a Pachyderm repository without downloading them first
+to your local computer. Pachyderm enables
+you to do so with the `pachctl mount` command. This command
+uses the SSH Filesystem (SSHFS) client and Filesystem
+in Userspace (FUSE) user interface to export a Pachyderm
+File System (PFS) to a Unix computer system. In other words,
+it allows you to mount a Pachyderm repository on your computer.
+
+For example, you have an [OpenCV pipeline](../../../getting_started/beginner_tutorial/#image-processing-with-opencv)
+up and running,
+and you need to edit files in the `images` repository. Let's
+say you want to experiment with brightness and contrast
+settings in `liberty.png`. Then, you want the pipelines
+run for the updated file and see the result in the end.
+If you do not mount the `images` repo, you would have to
+first download the files to your computer, edit them,
+and then put them back to the repository. The `pachctl mount`
+command automates all these steps for you. You can mount just the
+`images` repo or all Pachyderm repositories as directories
+on you machine, edit as needed, and, when done,
+exit the `pachctl mount` command. Upon exiting the `pachctl mount`
+command, Pachyderm uploads all the changes to the corresponding
+repository.
+
+## Avoiding Merge Conflicts
+
+If someone else modifies the files while you are working on them
+locally, their changes will likely be overwritten when you exit
+`pachctl mount`. This happens because your changes are saved to
+the Pachyderm repository only after you interrupt the `pachctl mount`
+command. Therefore, make sure that you do not work on the
```

Comment:
A few questions on the functionality here:

1. Isn't there a pachctl unmount command (or umount)? You don't need to kill or ctrl+C the mount, although that does work of course. I've had a number of issues with the unmount cmd failing (on Mac) so might need to do more testing here. 
2. If you use a writable mount and mess up your data and want to just "throw it away" without actually writing back to the repo, is there a way to do that? 
3. Does umount or killing the process give the user any warning or prompt before writing back to the repo. I could see these mount being fairly long lived and users even forgetting their mounts are writable and then making unintended commits. This is more of a product question for @jdoliner perhaps, but are there any reasonable ways we can protect users more here?

Reply:
1. there is no `pachctl unmount` command AFAIK. And it really just runs (block) in the terminal until you interrupt it. 
Of course, there are built-in UNIX commands that can mount/unmount your mounted dirs, which are literally `mount` and `unmount <disk>`. I don't know if we need to mention it in this doc. I presume it might be different on Ubuntu (`diskutil list` or similar).

2. Maybe @jdoliner  can answer this.
3. On macOS, none of this works. I don't think we have any protection against this

Reply:
We do have a `pachctl unmount` command, it's just a wrapper around the `umount` syscall, so I don't think there's any difference between calling `umount /pfs` from the command line, and calling `pachctl unmount /pfs` (except of course that we spell it correctly). The only advantage of `pachctl unmount` is that it will unmount multiple filesystems is you don't pass it an argument. I'm not sure how compelling the use case is for that, it only really applies if you're mounting a bunch of filesystems, which seems dubious, and then you kinda lose track of them and want to unmount them all to cleanup. Honestly the biggest reason for that behavior is it was useful to me during dev because I often wound up with a lot of mounts from running tests.